docs: Improve CONTRIBUTING.MD (#58)
flybywiresim/discord-bot-utils: .github/CONTRIBUTING.md

You can find the pull request template [here](PULL_REQUEST_TEMPLATE.md).

## Setting up the env

1. Copy the `.env.example` file and rename it to `.env`.
2. In your `.env` file replace `NODE_CONFIG_ENV=staging` with `NODE_CONFIG_ENV=development`.

> 3. Update the `TOKEN` with your own if you wish to use certain features.

Reviewer: I assume this, refers to updating TOKEN variables required for 3rd parties like the AVWX, WolframAlpha, etc.? If that's the case I would put it into a separate note under the `Setting up the bot` header.
```suggestion
```

### Setting up the config

1. Create a file named `development.json` at [../config/](../config/).
2. Paste the contents of [../config/staging.json](../config/staging.json) into `development.json`.
4. In your `development.json` file replace the `guildId` field with your server ID.
5. Update the channel and role IDs to those in your server.

> 6. Add your own user ID and that of your test bot to `modLogExclude `.

Reviewer 2: ```suggestion
6. Add your test bot ID to `modLogExclude ` and `userLogExclude`.
```

I didn't spot this in a previous PR, but lets change it here. It can be very useful to not have your user ID in the excludes, as it replicates production in this way. Then you can test things like scam logging, message edits and deletes. We also need to add the bot to user excludes so that we don't get a 'ping pong' of logs where the bot edits a message (edit message logs for example) then triggers another user log, getting itself in a loop.

> ### Setting up the env

Reviewer 2: Setting up the env should be a separate item. Not under 'Using different configurations'. The only overlap between the env and configs would be the `NODE_CONGIF_ENV`.

If wanting to have a separate entry on how to set up the env, I would suggest moving it to before the 'Using different configurations'. So there we tell the user to copy the env example, and update tokens if they wish to use certain features.

Then in the relevant sections for the config and setting up the bot we can keep the "In your `.env` file set `NODE_CONFIG_ENV` to `development`." and "Inside the .env file, type `BOT_SECRET=TOKEN` replacing TOKEN with what you copied in `step 6.`" respectively.

### Setting up the config

1. Create a file named `development.json` at [../config/](../config/).
2. Paste the contents of [../config/staging.json](../config/staging.json) into `development.json`.

> 3. In your `.env` file set `NODE_CONFIG_ENV` to `development`

Reviewer: Add a missing period.
```suggestion
3. In your `.env` file set `NODE_CONFIG_ENV` to `development`.
```

3. Give your application a name.
4. Navigate to the `Bot` tab and click `Add Bot`. You will have to confirm by clicking `Yes, do it!`.
5. Click the `Copy` button underneath token. (Do not share this).
6. Inside the `.env` file, find `BOT_SECRET=TOKEN` and replace the `TOKEN` with what you copied in `step 5`.

> 

Reviewer: Add the note about 3rd party token here.

```suggestion

> [!NOTE]
> If you want to use features that rely on 3rd parties like AVWX or WolframAlpha, you need to update their respective environment variables. See below for more information.

```
